sdks/common/runner-api/src/main/proto/beam_runner_api.proto

```diff
@@ -248,10 +248,20 @@ message Parameter {
 
 message StateSpec {
   // TODO: AST for state spec
+  string id = 1;
+  Type type = 2;
+
+  enum Type {
```

Comment:
Prefer top-level definitions, like `StateType`, so we don't couple names or generated classes.